doc/gallery/introduction/pytensor_intro.ipynb

```diff
@@ -0,0 +1,3311 @@
+{
```

Comment:
<p>Maybe cut this entirely, since these notebooks focus on pytensor itself, rather than PyMC's usage of it</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pytensor/pull/1585/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='108'/>

Reply:
Removed the seeding part and just left a small pymc model and show where PyTensor appears. Is this ok?

Reply:
<p>ok!</p>